toolchain/lowering/lowering_handle.cpp

```diff
@@ -220,10 +220,10 @@ auto LoweringHandleCall(LoweringFunctionContext& context, SemIR::NodeId node_id,
 
   if (llvm_function->getReturnType()->isVoidTy()) {
     context.builder().CreateCall(llvm_function, args);
-    // TODO: A function with a void return type shouldn't be referred to by
-    // other nodes.
+    // The value of a function call with a void return type shouldn't used, but
+    // StubReference needs a value to propagate.
```

Comment:
Maybe StubReference of `()` should be special-cased not to propagate?

Reply:
Yeah, I'll look into this in a follow-up.